### DISCUSSION TITLE
4.36.9.139 Firmware Download for v3

### DISCUSSION THREAD

**Discussion by Nerdtality** [gtxaspec/wz_mini_hacks#732] [General]
I have found the 4.36.9.139 for the v3 camera.

https://download.wyzecam.com/firmware/v3/demo_wcv3_4.36.9.139.zip

**Comment by Nerdtality** [comment]
Hmm, I see there is an archive of firmware available now. Wish I new that before I dumped Wyze firmware myself. Hahaha

[https://github.com/kohrar/Wyze-Firmwares](https://github.com/kohrar/Wyze-Firmwares)

Closing discussion!

**Reply by joebroke** [reply]
looks like it has a "Repository unavailable due to DMCA takedown." now.
